openeo/Pre&PostCovid.ipynb

```diff
@@ -0,0 +1,358 @@
+      "text/plain": [
+       "<Figure size 640x480 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax1 = plt.subplots(dpi=100)\n",
+    "ax1.plot(predata.t, predata.NO2.to_numpy().flatten(), color=\"g\")\n",
+    "ax1.set_xlabel(\"During COVID\")\n",
+    "ax2 = ax1.twiny()\n",
+    "ax2.plot(postdata.t, postdata.NO2.to_numpy().flatten(), color=\"r\")\n",
+    "ax2.set_xlabel(\"Post COVID\")"
```

Comment:
add a legend to the plot clearly indicating that the first (green) series is "During covid" and the second (red) is "Post covid"

Reply:
I would also add a y label "NO_2" or something like that

Reply:
labelled as NO2